test_goldens/testing/util.dart

```diff
@@ -114,10 +114,11 @@ Future<void> setUpBinding(
   Size size = mobileSize,
   Brightness brightness = Brightness.light,
 }) async {
-  tester.binding.window.physicalSizeTestValue = size;
-  tester.binding.window.devicePixelRatioTestValue = 1.0;
-  tester.binding.window.platformDispatcher.textScaleFactorTestValue = 1.0;
-  tester.binding.window.platformDispatcher.platformBrightnessTestValue =
+  tester.view.physicalSize = size;
+  tester.view.devicePixelRatio = 1.0;
+  tester.view.platformDispatcher.textScaleFactorTestValue = 1.0;
+  tester.view.platformDispatcher.platformBrightnessTestValue =
       brightness;
-  addTearDown(tester.binding.window.clearAllTestValues);
+  addTearDown(tester.view.reset);
+  addTearDown(tester.view.platformDispatcher.clearAllTestValues);
```

Comment:
Same here, you should be able to just use `tester.platformDispatcher`.

Reply:
Done.